Source/CarthageKit/Xcode.swift

```diff
@@ -1211,6 +1225,8 @@ public func buildInDirectory(directoryURL: NSURL, withConfiguration configuratio
 						}
 						return (project, containedSchemes)
 					}
+					// replace the filter and concat with something that accounts for multiple subprojects
+					// cause it errors out if even one of the results is that error...
```

Comment:
Unfortunately, I think this is going to be a prerequisite for merging this PR. Otherwise, we would likely break some projects that currently work with Carthage.

If you need help with this, I can try to steer you in the right direction.